sources/packages/forms/src/form-definitions/approvedeclineoffering.json

```diff
@@ -0,0 +1,489 @@
+            "tooltip": "",
+            "customClass": "",
+            "collapsible": false,
+            "hidden": false,
+            "hideLabel": true,
+            "disabled": false,
+            "modalEdit": false,
+            "key": "actionPanel",
+            "tags": [],
+            "properties": {},
+            "customConditional": "",
+            "conditional": {
+                "json": "",
+                "show": null,
+                "when": null,
+                "eq": ""
+            },
+            "logic": [],
+            "attributes": {},
+            "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+            },
+            "type": "panel",
+            "label": "Panel",
+            "breadcrumb": "default",
+            "tabindex": "",
+            "input": false,
+            "placeholder": "",
+            "prefix": "",
+            "suffix": "",
+            "multiple": false,
+            "defaultValue": null,
+            "protected": false,
+            "unique": false,
+            "persistent": false,
+            "clearOnHide": false,
+            "refreshOn": "",
+            "redrawOn": "",
+            "tableView": false,
+            "dataGridLabel": false,
+            "labelPosition": "top",
+            "description": "",
+            "errorLabel": "",
+            "autofocus": false,
+            "dbIndex": false,
+            "customDefaultValue": "",
+            "calculateValue": "",
+            "calculateServer": false,
+            "widget": null,
+            "validateOn": "change",
+            "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+            },
+            "allowCalculateOverride": false,
+            "encrypted": false,
+            "showCharCount": false,
+            "showWordCount": false,
+            "allowMultipleMasks": false,
+            "addons": [],
+            "tree": false,
+            "lazyLoad": false,
+            "components": [
+                {
+                    "label": "Description",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "tag": "p",
+                    "className": "",
+                    "attrs": [
+                        {
+                            "attr": "",
+                            "value": ""
+                        }
+                    ],
+                    "content": "{{data.offeringStatus === 'Approved' ? 'Outline the reasoning for approving this request.  This will be stored in the institution profile notes.' : 'Outline the reasoning for declining this request.  This will be stored in the institution profile notes.'}}",
```

Comment:
is there extra space in between `request.  This`

Reply:
This component itself is changed. Btw the extra space is removed.